### PR TITLE
added implementation for RAG to get doctor recommendations

### PR DESCRIPTION
**Pull request by coderkj7**

### CODE REVIEW

File: src/mediwise_chatbot/utils.py [hunk 1]
```diff
@@ -188,7 +252,12 @@ def tool_call(messages, response_message, tool_calls):
                 function_response = function_to_call(
                     doctor=function_args.get("doctor"),
                     )
-
+            elif function_name == 'table_dml':
+                function_response = function_to_call(
+                    dml=function_args.get("dml"),
+                    )
+            print(function_response)
+            print(function_response)
```
Comment (lizardgai4) [inline src/mediwise_chatbot/utils.py:260]: Do we need to print it twice?

**Review by lizardgai4 — APPROVED** [review]
All these look good.  The only thing that doesn't look good is two identical print statements next to each other

**Comment by nady717** [comment]
let me check one more time.. I did some changes late last night.. will update you

File: src/mediwise_chatbot/utils.py (hunk 1 above)
Comment (coderkj7) [reply src/mediwise_chatbot/utils.py:260]: Prints are not needed. We can do the code cleanup at a later time as long as the changes are not breaking any existing functionality then we should be ok to merge

Comment (lizardgai4) [reply src/mediwise_chatbot/utils.py:260]: Got it

**Pull request merged**